docs/release-notes/release-notes-0-17-10.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Release Notes 0.17.10
+sidebar_position: 99
+---
+
+This document describes what has changed in Macrometa release 0.17.10. All accounts will be updated by August 18, 2022. To check what version of GDN you are using, use the [Client Version](https://macrometa.com/docs/cli/client-version-cli.md) command in the CLI or the [RestVersionHandler](https://macrometa.com/docs/api#/operations/RestVersionHandler) command in the API.
+
+## Features
+
+The following features are included in this release.
+
+### UI Updates
+
+This release included a complete overhaul of the Macrometa UI. A few section names changed, but no overall functionality changed as a result of this new feature.
+
+- Updated the overall look and feel of the web console UI.
+- Changed order of items in the side menu.
+- User icon and log in/out and settings moved from the upper right to lower left corner.
+- SEARCH changed to Search Views.
+- QUERIES changed to Query Workers.
+- Query Profile changed to Query Info in Query Results.
+- **Download CSV** temporarily removed from Query Results. This feature will return when we have a better CSV solution.
+
+Various screenshots and instructions were updated to reflect the new UI.
+
+![Redesigned UI](/img/release-notes/17-10-new-ui.png)
+
+### Billing Updates
+
+This release includes an overhaul of the billing functionality, including:
+
+- New invoice design.
+- Redesigned billing tab, with graphs and metrics to help you understand usage.
+- Various performance improvements.
+- A bunch of billing-related defect fixes on the backend.
+
+If you have a paid Macrometa plan, then you can view the new billing screen in your account Overview tab.
+
+![New billing tab](/img/release-notes/17-10-new-billing-tab.png)
+
+### Kafka Enabled on Macrometa
+
+Kafka is now enabled on Macrometa.
+
+### Update Name and Email on User Accounts
+
+You can now update names and email addresses on user accounts. If you are a `root` user or have admin permissions, then you can update names and email addresses on user accounts that you administer.
+
+### API Updates
+
+This release includes the following API changes.
+
+| Issue #  | Description  |
+|---|---|
+| API-178  | Add region URL to the API response.  |
+| DB-1500  | Add new database limits: `maxWorkersCpuSecondsPerMinute` (default value is 20 seconds/minute of CPU time), `maxWorkersLogsLengthKBPerMinute` (default value is 10 KB/minute of logs length).  |
+| DB-1588  | Removed one fabric parameter from Users API, because having two parameters was confusing. Now consistent with other APIs.  |
```

Comment:
```suggestion
| DB-1588  | Removed one fabric parameter from Users API because having two parameters was confusing. Now consistent with other APIs.  |
```